src/api/connection.ts

```diff
@@ -321,6 +330,36 @@ export class SimpleConnection implements Connection {
         }));
     }
 
+    /**
+     * @inheritdoc
+     */
+    public getBinary(path: string, parameters: {} = {}): Rx.Observable<ArrayBuffer> {
+        return Rx.Observable.create((observer) => {
+            const url = this.createUriFromPath(path, parameters);
+
+            const xhr = new XMLHttpRequest();
+            xhr.open("GET", url, true);
+            xhr.responseType = "arraybuffer";
+            xhr.withCredentials = true;
+
+            xhr.onreadystatechange = () => {
+                if (xhr.readyState === XMLHttpRequest.DONE) {
+                    if (xhr.status === 200) {
+                        observer.onNext(xhr.response);
+                    } else {
+                        observer.onError(xhr.response);
+                    }
+                }
+            };
+
+            xhr.send(null);
```

Comment:
Inspired by https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Sending_and_Receiving_Binary_Data.

jQuery doesn't seem to support `arrayBuffer` response type out of the box.